Files/MultilingualResources/Files.de-DE.xlf

```diff
@@ -1968,18 +1953,15 @@
         </trans-unit>
         <trans-unit id="PreviewPaneToggle.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
           <source>Toggle the preview pane (Ctrl+P)</source>
-          <target state="needs-review-translation">Vorschaupanel anzeigen (Ctrl + P)</target>
-          <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
+          <target state="translated">Vorschau anzeigen (Strg + P)</target>
```

Comment:
This translation is not ideal since we also show this tooltip when the pane is open. How about 
```suggestion
          <target state="translated">Vorschau anzeigen/ausblenden (Strg + P)</target>
```